lumigator/python/mzai/schemas/lumigator_schemas/jobs.py

```diff
@@ -105,3 +104,7 @@ class JobResultResponse(BaseModel, from_attributes=True):
 class JobResultDownloadResponse(BaseModel):
     id: UUID
     download_url: str
+
+
+class Job(JobResponse, JobSubmissionResponse):
```

Comment:
What is the rationale behind merging together all info? I am trying to understand whether it'd make sense to have a `ray` subsection here that might as well be optional if there's no job info on ray rather than a bunch of null fields.

Reply:
The rationale was mostly to reduce any changes required by consuming code as there weren't really any issues with conflicting fields (only status). Code that was calling `/health/jobs` related endpoints could change it's endpoint to `/jobs` (removing `/health`) and get the same data structure as before for the job (with a few additional ones like `name` and `description`).

This felt like a sensible middle ground until we have experiments in place and a more well defined set of models/boundaries etc for our API. So a step in the right direction, but not the last step. 😄